src/KM_Controls.pas

```diff
@@ -6876,20 +6952,21 @@ procedure TKMMasterControl.UpdateFocus(aSender: TKMControl);
     begin
       // If there was no focus, or it is our Focus control that went hidden or disabled
       CtrlFocus := nil;
-      FindFocusable(fCtrl);
+      FindFocusable(fMasterPanel);
     end;
   end;
 end;
 
 
 //Check if control is covered by other controls or not
-//We assume that control is covered if any of his 4 angles is covered
+//We assume that control is covered if any of his 4 corners is covered
+//For corners used actual corners with 1 px offset inside to solve border collisions
 //Use Self coordinates to check, because some controls can contain other sub-controls (f.e. TKMNumericEdit)
 function TKMMasterControl.IsCtrlCovered(aCtrl: TKMControl): Boolean;
 begin
-  Result := (HitControl(aCtrl.SelfAbsLeft, aCtrl.SelfAbsTop) <> aCtrl)
-        or (HitControl(aCtrl.SelfAbsLeft + aCtrl.SelfWidth - 1, aCtrl.SelfAbsTop) <> aCtrl)
-        or (HitControl(aCtrl.SelfAbsLeft, aCtrl.SelfAbsTop + aCtrl.SelfHeight - 1) <> aCtrl)
+  Result := (HitControl(aCtrl.SelfAbsLeft + 1, aCtrl.SelfAbsTop + 1) <> aCtrl)
+        or (HitControl(aCtrl.SelfAbsLeft + aCtrl.SelfWidth - 1, aCtrl.SelfAbsTop + 1) <> aCtrl)
+        or (HitControl(aCtrl.SelfAbsLeft + 1, aCtrl.SelfAbsTop + aCtrl.SelfHeight - 1) <> aCtrl)

```

Comment:
fix bug with borders on 'Self' coordinated - should not only substract 1 px (-1), but lso add 1px (+1)